docs/configuration/modules/remote-service.md

```diff
@@ -31,6 +31,28 @@ to connect to the remote server. The default value of this property is `true`.
 The `port` property is an optional property that defines the connection port. By default, if SSL/TLS protocol
 is used, the default value of this property is `443`. Otherwise, it is `80`.
 
+## autoreconnect-enabled
+
+The `autoreconnect-enabled` property is an optional property that defines if auto reconnection to the server should be used when to disconnect. The default value of this property is `false`.
+
+## reconnection-initial-interval
+
+The `reconnection-initial-interval` property is an optional property that defines the reconnection initial interval. The default value of this property is `5`in seconds.
+
+## reconnection-interval-mutiplier
+
+The `reconnection-interval-mutiplier` property is an optional property that defines the reconnection interval mutiplier. The default value of this property is `2`.
+
+## reconnection-max-interval
+
+The `reconnection-max-interval` property is an optional property that defines the max interval between two reconnection. The default value of this property is `3600`in seconds.
+With the default parameters, there would be no reconnection attempt.
+With autoreconnect set to true and the other parameters set to default values, in the case of a disconnection, the reconnection attempts would occur after 5", then 10", 20", 40", 1'20", 2'40" .... etc incresing until a maximum of one hour interval. after reconnection max intenal, the reconnection will be re-attempted every hour.
```

Comment:
*With `autoreconnect-enabled` set to `true`

Reply:
It's done